src/algoliasearch.helper.js

```diff
@@ -306,10 +305,15 @@ AlgoliaSearchHelper.prototype.searchForFacetValues = function(facet, query, maxF
 
     content = Array.isArray(content) ? content[0] : content;
 
-    content.facetHits = forEach(content.facetHits, function(f) {
-      f.isRefined = isDisjunctive ?
-        state.isDisjunctiveFacetRefined(facet, f.value) :
-        state.isFacetRefined(facet, f.value);
+    content.facetHits = Array.isArray(content.facetHits)
```

Comment:
For which conditions `facetHits` is not an array?

Reply:
Some of the tests return an empty object, but indeed I don't think the API can do that 🤔

Reply:
We should fix the tests then.

Reply:
https://github.com/algolia/algoliasearch-helper-js/pull/686